open-craft/xblock-poll
# [BD-29] [TNL-6993] [BB-2706] Make Poll and Survey Indexable

Make Poll and Survey Indexable

**JIRA tickets**: 
* https://openedx.atlassian.net/browse/TNL-6993

**Merge deadline**: "None"

**Testing instructions**:

1. Clone this repo in `src` directory of your devstacks.
2. Checkout this branch i.e `ahmed/bb-2706-make-poll-and-survey-indexable`
3. Install the cloned project into both **LMS** and **Studio** by changing your directory to the `src/xblock-poll` directory in your devstack and executing `pip install -e .`
4. Add `poll` and `survey` to the Advanced Module List. See https://github.com/open-craft/xblock-poll#setting-up-a-course-to-use-polls-and-surveys
5. Create two blocks. One poll and other survey.
6. Reindex your course.
7. Go to LMS and search for some string that appears in the poll or survey

**Reviewers**
